helios-services/src/main/java/com/spotify/helios/agent/QueueingHistoryWriter.java

```diff
@@ -272,12 +305,13 @@ public void run() {
         return;
       }
 
+      final JobId jobId = item.getStatus().getJob().getId();
+      final String historyPath = Paths.historyJobHostEventsTimestamp(
+              jobId, hostname, item.getTimestamp());
+      log.debug("writing queued item to zookeeper {} {}", item.getStatus().getJob().getId(),
```

Comment:
Same reason you have the `log.debug` for sending an event to Kafka in that block. Just to put the log statement next to where that is actually happening, right?